src/docker/DockerCompose.js

```diff
@@ -129,6 +129,79 @@ class DockerCompose {
     }
   }
 
+  /**
+   * List all docker compose containers with status
+   *
+   * @param {string} preset
+   * @return {Promise<void>}
+   */
+  async listContainerStatus(preset) {
```

Comment:
`inspectService(preset, serviceName = undefined)`?

It seems like we could reuse the code from `isServiceRunning`. You could extract duplicate logic to the private method and call it in both methods.

Reply:
Do you mean I should create a private method using the new `#listContainers()` syntax described [here](https://github.com/tc39/proposal-private-methods#private-methods-and-fields)? This would bump our Node version requirement from 10 to 12 I think. Or should I just create a normal method that is not used outside the `DockerCompose` class?

Reply:
Yeah, just private method for `DockerCompose`. JS doesn't support private methods so you can just mar it as a private using JSDoc

Reply:
Yep, bump minimal version if you need it.

Reply:
Actually I got confused since there is a TC39 proposal for true private methods and fields in JS which is supported in Node 12. But I've just followed the style of the other methods in the class for now.